### PR TITLE
App Store

### PR DESCRIPTION
**Pull request by justnero**
[Preview link](https://preview.redoc.ly/rebilly-users/app-store/tag/Application-owners/#operation/PostApplication)

### CODE REVIEW

File: openapi/components/schemas/Applications/Application.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      - $ref: ../ServerTimestamp.yaml
+  _links:
+    type: array
+    description: The links related to resource.
+    readOnly: true
+    minItems: 1
+    items:
+      anyOf:
+        - $ref: ../Links/SelfLink.yaml
+        - $ref: ../Links/ApplicationInstanceLink.yaml
```
Comment (rensftw) [inline openapi/components/schemas/Applications/Application.yaml:76]: I think we are missing `../Links/LogoUrlLink.yaml` as an option here?

Comment (rensftw) [reply openapi/components/schemas/Applications/Application.yaml:76]: Oh, nevermind - my bad.

**Comment by rensftw** [comment]
Do we need to update `openapi/components/schemas/Files/File.yaml` to have `./openapi/components/schemas/Links/LogoUrlLink.yaml` as an option?

File: openapi/components/schemas/Applications/Application.yaml (hunk 1 above)
Comment (rensftw) [reply openapi/components/schemas/Applications/Application.yaml:76]: 🤔 Thinking again about this.. 
If we had this, then it saves us one BE call from Recomm

Comment (n-fasano) [reply openapi/components/schemas/Applications/Application.yaml:76]: You're right, I added it

**Review by rensftw — APPROVED** [review]

**Review by VladimirBerdnik — APPROVED** [review]

**Review by n-fasano — APPROVED** [review]

**Pull request merged**